package.json

```diff
@@ -17,9 +18,9 @@
     "prebuild": "npm run format && npm run lint",
     "build": "rm -rf dist && tsc",
     "postbuild": "shx chmod +x dist/*.js",
-    "dev": "tsx . run",
-    "dev:debug": "DEBUG=auth0-mcp tsx . run",
-    "dev:inspect": "npx @modelcontextprotocol/inspector tsx . run",
+    "dev": "tsx src/index.ts run",
```

Comment:
This is needed otherwise it uses what's in main (dist/index.js) and thus has no effect.